src/test/java/pokemanager/StoreCommandTest.java

```diff
@@ -0,0 +1,36 @@
+package pokemanager;
+
+import org.junit.Test;
+
+import java.io.*;
+
+import static org.junit.Assert.*;
+
+public class StoreCommandTest {
+
+    private final ByteArrayOutputStream out = new ByteArrayOutputStream();
+    private final PrintStream pw = new PrintStream(out);
```

Comment:
I think the relationship between the variable name `pw` and what it actually is (a PrintStream) is not quite clear! Maybe call it `printStream`...